### PR TITLE
docs: Update version : git checkout tags/v2.2.1 to v2.19.1

### PR DESCRIPTION
**Pull request by t-leclercq**
On a brand new system (Ubuntu), composer is facing issues while browsing all packages through PackageManifest.php
Checking the latest Monica version solves most of the composer issues.

==> I've read the PR Guidelines but as it is just a documentation update request, no branch to merge.

### CODE REVIEW

**Review by t-leclercq — COMMENTED** [review]
docs: change version

File: docs/installation/providers/ubuntu.md [hunk 1]
```diff
@@ -97,8 +97,9 @@ You should check out a tagged version of Monica since `master` branch may not al
 
 ```sh
 cd /var/www/monica
-git checkout tags/v2.2.1
+git checkout tags/v2.19.1
 ```
+// See the last release version : https://github.com/monicahq/monica/releases (15 Feb 2021 update : 2.19.1 at the moment)
```
Comment (asbiin) [inline docs/installation/providers/ubuntu.md:102]: This cannot stay here.
Maybe your intention was to add it as a comment? There is no such thing in markdown...

**Review by asbiin — CHANGES_REQUESTED** [review]
Thank you for you help @t-leclercq 
However, I don't understand the value of it. Users will have to watch for the last release in the release page, and we have no intention of updating this doc (and others with the same command) each time a release goes out!

File: docs/installation/providers/ubuntu.md (hunk 1 above)
Comment (t-leclercq) [reply docs/installation/providers/ubuntu.md:102]: I'm removing it.
This was to trigger the discussion but - yeah - this has no place here.

**Comment by t-leclercq** [comment]
Hi @asbiin ! :)

> Users will have to watch for the last release in the release page

This was to avoid users to watch the releases, a simple update in each stable release push would probably do the trick once and for all.
I'll get it if there is no time to devote to this UX task.

**Comment by asbiin** [comment]
Thank you @t-leclercq for your contribution.
We always like to see the community participate in this great project.
Unfortunately, we will not merge this proposal.

Version will be increase in the future, and we don't want to update these documentation - and all documentation regarding `git checkout` action each time a release is shifted. People should check the release page to checkout the proper version.

I hope you'll understand.

Thank you!

**Pull request closed without merging**